src/software/ai/navigator/path_planner/theta_star_path_planner.h

```diff
@@ -121,73 +132,72 @@ class ThetaStarPathPlanner : public PathPlanner
 
        private:
         Coordinate parent_;
-        double f_, g_;
+        double path_cost_and_end_dist_heuristic_;
+        double best_path_cost_;
         bool initialized_;
     };
 
     /**
      * Returns if a cell is within bounds of grid
      *
-     * @param test_coord Coordinate to consider
+     * @param coord Coordinate to consider
      *
-     * @return true if cell is valid
+     * @return true if cell is navigable
      */
-    bool isCoordValid(Coordinate test_coord);
+    bool isCoordNavigable(const Coordinate &coord) const;
 
     /**
      * Returns if a cell is unblocked
      *
-     * @param test_coord Coordinate to consider
+     * @param coord Coordinate to consider
      *
      * @return true if cell is unblocked
      */
-    bool isUnBlocked(Coordinate test_coord);
+    bool isUnblocked(const Coordinate &coord);
 
     /**
-     * Returns heuristic value of a cell,
-     * currently the Euclidean distance to the destination
+     * Computes distance from coord1 to coord2
```

Comment:
nitpicking here because this is a graph search algorithm, can we specify this is Euclidian distance?